docs/01-nodejs-context.md

```diff
@@ -0,0 +1,109 @@
+# How do Node.js Contexts works
```

Comment:
Should be :
- How do Node.js Contexts Work
- How does Node.js Context Work
Could be:
- Understanding Node.js Asynch Context

Reply:
Fixed! https://github.com/KTH/skog/pull/16/commits/53871fab14bd5c2b076f4ab733813d2c9db1ccc4